### DISCUSSION TITLE
Using typescript

### DISCUSSION THREAD

**Discussion by chrisbc** [GNS-Science/simple-toshi-ui#7] [General]
@wooneh I've done some reading and think we shouls proceed with ts in the project as you suggeat. I raised  #5 for this.

I'm sure I'll have a few questions so I'm setting up this discussion thread in advance.

**Comment by wooneh** [comment]
Great! Would you like me to pick up that ticket?

Edit: Just noticed it was assigned to me :)

**Reply by chrisbc** [reply]
👍🏽 cool yep  it's yours :)
